cirq-core/cirq/sim/act_on_density_matrix_args.py

```diff
@@ -35,7 +35,9 @@ class ActOnDensityMatrixArgs(ActOnArgs):
 
     def __init__(
         self,
-        target_tensor: np.ndarray,
+        target_tensor: Optional[np.ndarray] = None,
+        initial_state: Union[np.ndarray, 'cirq.STATE_VECTOR_LIKE'] = 0,
```

Comment:
Oh, I was intending that we extend the type of `target_tensor` to accept ints etc too, not introduce a new parameter. (Same in other simulators).

Reply:
I see... that is a much better way than adding a new parameter!

Reply:
Thanks, you can resolve this one.